README.md

```diff
@@ -1,37 +1,162 @@
-# Onfido OpenAPI specification (beta)
+# Onfido OpenAPI specification
 
-:warning: Our OpenAPI specification is currently in beta and we welcome any feedback. You can contact us via the issues tab or [support portal](https://public.support.onfido.com), but we don't yet officially support this specification. :warning:
+## Introduction
 
-This specification supports the latest version of the Onfido API.
+This specification supports the latest version of the Onfido API (v3.6).
 
-For our latest stable release that is officially supported please use [v1.0.0](https://github.com/onfido/onfido-openapi-spec/tree/v1.0.0).
+It is also used for generating client libraries to allow backend services to interact with Onfido API. A postman collection is provided as well for user's convenience.
```

Comment:
```suggestion
It can also be used for generating client libraries to allow backend services to interact with the Onfido API. A [Postman collection](https://god.gw.postman.com/run-collection/221325-406603c0-7f64-4251-a58c-93324d99f916) is provided as well for user's convenience.
```

Reply:
I'd just not include the postman collection link here in this summary section, also because it's going to change (at least for now). But we've the link later on in the documentation portal section pointing to postman later on.

Reply:
Sure, thank makes sense

Reply:
I'll amend that later on, adding a link to the section pointing to the Postman collection.